Kobold Press; Deep Magic Backgrounds, Feats, & Subclasses

TryHardSatan opened the pull request:
Transcribed Kobold Press's Deep Magic: Backgrounds, Feats & Subclasses from Roll20 to be useable in FoundryVTT
Contains:
4 Backgrounds
2 Feats
6 Items (All related to Alkemancer Subclass)
55(56*) Subclasses (* Angelic Scribe has been created twice for both Wizard & Cleric)
All Subclass features & Optional features included

hakr14 commented on collection/Kobold Press; Deep Magic Backgrounds, Feats, & Subclasses.json:
```diff
@@ -0,0 +1,11555 @@
+				{
+					"choose": {
+						"from": [
+							"Elvish",
+							"Sylvan"
```
These should be lowercase.

hakr14 commented on collection/Kobold Press; Deep Magic Backgrounds, Feats, & Subclasses.json:
```diff
@@ -0,0 +1,11555 @@
+				{
+					"choose": {
+						"from": [
+							"Infernal",
+							"Abyssal"
```
My mistake. These (and all languages in "languageProficiencies" objects, for future reference) should also be lowercase.

TryHardSatan replied:
Thank you! 
Its the first time ive made a compendium let alone a JSON file (or even touched coding) so this is all quite new to me.
I'll make that change and apply it, if you need me to change more to make it work please let me know!

TheGiddyLimit reviewed (changes requested):
fwiw you can go to the "Checks" tab here on Github and see exactly why the file is invalid. Then, if you're struggling to understand the error messages, do feel free to ask for an explanation.

TheGiddyLimit reviewed (changes requested):
you can also run these tests locally if you have NPM installed, then do an `npm i` into an `npm t`

TryHardSatan commented:
> you can also run these tests locally if you have NPM installed, then do an `npm i` into an `npm t`

How do I do that? is npm something I can run in Visual Studio?
I have never coded anything in my life before so this is all magic to me.

But i think I've fixed it - i think its the Languages part being capitalised - I've changed everything I can think of to be correct but if that doesn't work I may need some help

TheGiddyLimit commented:
you'd download and install Node.js: https://nodejs.org/en/
then run those commands in your homebrew directory in your terminal of choice (the terminal in VS Code will do just fine!)

p4535992 commented:
@TryHardSatan  Here the file corrected  file it was a small thing, just three language tags to fix (the file is renamed with ".txt" for github limitaitons)
[Kobold Press; Deep Magic Backgrounds, Feats, & Subclasses.json.txt](https://github.com/TheGiddyLimit/homebrew/files/9834070/Kobold.Press.Deep.Magic.Backgrounds.Feats.Subclasses.json.txt)

TheGiddyLimit commented:
Duplicate of #1518

TryHardSatan commented:
Thank you so much!

Im really Sorry, i was busy with a new job and didnt have time to go over this - thank you both @TheGiddyLimit & @p4535992 
Myself and others can hopefully enjoy the Kobold Press Classes in their games now!  :D
